contrib/hooks/README.rst

```diff
@@ -0,0 +1,20 @@
+.. image:: https://easybuilders.github.io/easybuild/images/easybuild_logo_small.png
+   :align: center
+
+`EasyBuild <https://easybuilders.github.io/easybuild>`_ is a software build
+and installation framework that allows you to manage (scientific) software
+on High Performance Computing (HPC) systems in an efficient way.
+
+The **easybuild-framework** package is the core of EasyBuild. It
+supports the implementation and use of so-called easyblocks which
+implement the software install procedure for a particular (group of) software
+package(s).
+
+The EasyBuild documentation is available at http://easybuild.readthedocs.org/.
+
+The EasyBuild framework source code is hosted on GitHub, along
+with an issue tracker for bug reports and feature requests, see
+https://github.com/easybuilders/easybuild-framework.
```

Comment:
@akesandgren Most of this stuff above is distracting?

I would just include a link to the EasyBuild website + a short description of what this dir is about + a link to the hooks documentation @ https://easybuild.readthedocs.io/en/latest/Hooks.html .

Maybe we should also include some instructions on how to add your own hooks (maybe in subdirs with a descriptive name?)

Reply:
And with the easybuild web site you explicitly mean what URL?

Reply:
I would say https://easybuild.readthedocs.io

Reply:
Something like that then?

Reply:
@boegel Never got any reply on this last change.